packages/host/tests/acceptance/code-submode/inspector-test.ts

```diff
@@ -905,7 +905,7 @@ module('Acceptance | code submode | inspector tests', function (hooks) {
   });
 
   test<TestContextWithSave>('can duplicate an instance in different realm', async function (assert) {
-    assert.expect(11);
+    assert.expect(8);
```

Comment:
an unnecessary save was being issued which doubled up the onSave assertions, fixed.